### ISSUE TITLE
Probing experiments

### ISSUE THREAD

**Issue by odunkel** [alibaba/EasyNLP#356]
Hello,

thanks for the very interesting work 'Towards Understanding Cross and Self-Attention in Stable Diffusion for Text-Guided Image Editing'.
Do you also plan to release the code, trained classifiers etc. for performing the probing analysis?

Thanks for your support in advance.

**Comment by Bingyan-Liu** [comment]
Thank you for your attention. Unfortunately, for the Probing experiments, we validated on the test data directly after each training session without saving the trained classifier. The classifier is a two-layer MLP as follows, which can be easily trained using torch once the data is prepared.

class Net(torch.nn.Module):
    def __init__(self, input_size: int = 64 * 64, num_classes: int = 10):
        super(Net,self).__init__()
        self.input_size = input_size
        self.hidden_size = max(input_size // 8, 2048)
        self.mlp = torch.nn.Sequential(
            torch.nn.Flatten(),
            torch.nn.Dropout(0.1),
            torch.nn.Linear(in_features=self.input_size, out_features=self.hidden_size),
            torch.nn.Tanh(),
            # torch.nn.ReLU(),
            torch.nn.Linear(in_features=self.hidden_size, out_features=num_classes)
        )
    def forward(self, x):
        output = self.mlp(x) 
        return output

**Comment by shaswati1** [comment]
@Bingyan-Liu, thanks for this awesome work! Which timestep(s) did you consider for probing experiments? And, how many samples did you use to train the classifier?

**Comment by Bingyan-Liu** [comment]
> @Bingyan-Liu, thanks for this awesome work! Which timestep(s) did you consider for probing experiments? And, how many samples did you use to train the classifier?

We took the average over all steps (50 steps) . We sampled 2000 examples, 80% for training and 20% for testing.
